### ISSUE TITLE
when reorder Maps on the toolbar it does not save location

### ISSUE THREAD

**Issue by jeremia90** [toopriddy/mytime#311]
Hi Brother,

if I want to reorder the signs in the bottom and I chose not to have the "Map" sign but rather for instance the "statistics", everytime I start the app again: the "Map" is here again and nothing has changed.

I dont need the Map function at all but rather neeeeeeeed the statistics to enter quickly and manually some rv or hours.

Can you help or fix?

Your Brother from Germany
Lukas

**Comment by toopriddy** [comment]
Hello Lukas, Please make sure that you do not close MyTime by force
quitting the application using the multitasking bar.  MyTime uses the time
that you put the application into the background to save user configuration
like the tab bar button ordering.

FYI: force closing a currently in focus application is super bad, it will
force quit an application even if it is writing to the disk and will most
assuredly corrupt your data.

Force quitting applications is almost never needed and sadly many people
have been indoctrinated by android into doing this on the iPhone.  The
iPhone does this for you automatically when it is safe to do so and it is
never needed that a user do this themselves.  It does not save battery it
actually makes the battery drain more because of having to re-initalize
applications every time you start them.

For MyTime, if you force close the app after rearranging the tab bar
buttons, it will not save your preferences. just rearrange and tap on the
home button and it will be saved.

I hope this helps.

**Comment by Appaljack** [comment]
Brent, not true on the Camera app. The iPhone gets quite warm and runs the battery down unless you swipe it out of RAM. I believe this might be the gps and metadata stuff running on it but after a suitable length of time, we close all the apps we've opened. Some, like ones with ads or tracking, will send data unless you use Airplane mode. They run in the background continuously (or can be made to do so like some of the 'flashlight' apps). Take care, bro.

**Comment by toopriddy** [comment]
To try to be helpful:

You might want to check what is going on, if you have a jailbroken
application or a hardware problem you will see what you are talking about
for the camera application but this is simply not the normal case (normally
the problem is if people do not turn off their phone and put the phone with
the camera app on into their pocket).
The camera app does not do GPS and metadata while in the background.

Non apple applications with ads or tracking are not allowed to run in the
background for more than 5-10 seconds and normally apple kills them in the
approval process (I know).  any questionable application that you download
which plays a sound to keep running in the background is getting killed by
apple and lets you know that it is doing this by a red banner.

For whatever it is worth, only force killl applications while they are in
the background (you are looking at the home screen and then double tap the
home button to then kill apps). if you kill an application in the
foreground you are pulling the plug and it is disastrous for the
application and your data that it is trying to write to the disk.  I have
for the last 8 years tried to get people to quit doing this android
behavior, even apple says that this is not needed but still people do it.

**Comment by Appaljack** [comment]
No, have never jailbroke any phone and it has done it on three iPhone6+'s and two new iPhone7+'s. That makes it highly unlikely it is hardware (most likely a software problem which is not unusual since it is an Apple app). When we swipe it off on a Home double-press, it cools the phone. As I said, waiting for it to finish is fine and we swipe off and reboot about once weekly or things just start wonking out. Apple has also managed to lose people's data on major updates by moving the file location identifiers. But don't worry, Windows has a far worse track record in my book. And my daughter continuously fools with Android, lol.

**Comment by Appaljack** [comment]
Also, the native camera app is actually accessible on thevlock screen and instantly pops up but this can be changed, I believe.

**Comment by toopriddy** [comment]
Lukas, please let me know if this resolves your issue so that I can close
this issue report

**Comment by jeremia90** [comment]
Brent, I'll try today in my ministry and tell you if it works. Sounds logical to me. I already know about that issue but didnt connect it to this problem. :-)

**Comment by jeremia90** [comment]
Your solution helped for some hours, but after opening it some hours later, same problem. I didn't close it via task manager. Just used different apps. :-(

The interesting part: The other symbols I changed are always saved, no matter how I close the app. The problem is only with the Maps Sign.

**Comment by toopriddy** [comment]
validated that this works on the latest
